doc/exception_interrupts.rst

```diff
@@ -8,10 +8,10 @@ Ibex implements trap handling for interrupts and exceptions according to the RIS
 All exceptions cause the core to jump to the base address of the vector table in the ``mtvec`` CSR.
 Interrupts are handled in vectored mode, i.e., the core jumps to the base address plus four times the interrupt ID.
 
-The base address of the vector table is given by the boot address (must be aligned to 256 bytes, i.e., its least significant byte must be 0x00).
-The most significant 3 bytes of the boot address given to the core are used for the first instruction fetch of the core and as the basis of the interrupt vector table.
+The base address of the vector table is initialized on booting the core.
+The most significant 3 bytes of the boot address given to the core are used for the first instruction fetch of the core and to initialize the base of the interrupt vector table when the core is booted.
 The core starts fetching at the address made by concatenating the most significant 3 bytes of the boot address and the reset value (0x80) as the least significant byte.
-The boot address can be changed after the first instruction was fetched to change the interrupt vector table address.
+It is possible to change the base address after booting using the instructions added by the Zicsr extension.
```

Comment:
It is possible to change the base address after booting through the ``mtvec`` CSR.